fossunited/public/css/custom.css

```diff
@@ -330,7 +330,7 @@ h6{
 
 .primary-button{
     display: flex;
-    padding: 0.625rem 0.75rem;
```

Comment:
Why? These are the dimensions which are in design. Please attach a screenshot for before and after of this.